heartbeat/monitors/pluginconf.go

```diff
@@ -28,6 +28,7 @@ var ErrPluginDisabled = errors.New("Monitor not loaded, plugin is disabled")
 
 // MonitorPluginInfo represents the generic configuration options around a monitor plugin.
 type MonitorPluginInfo struct {
+	ID      string `config:"id"`
```

Comment:
Shouldn't name also show up here?

Reply:
Good point, we can DRY up the code by doing so. We actually reimplement this once per monitor at the moment!